topo/node/cptx/cptx_test.go

```diff
@@ -129,9 +130,15 @@ func TestConfigPush(t *testing.T) {
 			}
 			defer fp.Close()
 
+			/* Using scrapl_cfg_testing results in an EOF error when config includes comments.
+			 * We remove the first line of the sample config, which holds the plaintext password.
+			 * Comments in config files are not problematic when using kne_cli. */
+			fbuf := bufio.NewReader(fp)
```

Comment:
this is a simple solution which works for this case, my concern is that it will cause issues if the test cases changes in the future

it is safer here to write a small helper func which removes comment lines from a config and returns the comment-free version